crates/loader/Cargo.toml

```
@@ -32,3 +32,8 @@ tokio-util = "0.6"
 toml = "0.5"
 tracing = { workspace = true }
 walkdir = "2.3.2"
+
+[features]
+default = ["local"]
```

Let's make the default backward-compatible for now:

```suggestion
default = ["bindle", "local"]
```